### ISSUE TITLE
Cannot build recent superset-ui on certain environments

### ISSUE THREAD

**Issue by b2kdaman** [apache-superset/superset-ui#226]
**Describe the bug**
I'm trying to build different versions on my Ubuntu environment, And I always get Typescript driver failure.

**To Reproduce**
yarn build
yarn install
lots of errors

**Command failed with exit code 1 (EPERM): tsc --emitDeclarationOnly

Command failed with exit code 1 (EPERM): tsc --emitDeclarationOnly

Command failed with exit code 1 (EPERM): tsc --emitDeclarationOnly

Command failed with exit code 1 (EPERM): tsc --emitDeclarationOnly

Command failed with exit code 1 (EPERM): tsc --emitDeclarationOnly

Command failed with exit code 1 (EPERM): tsc --emitDeclarationOnly

Command failed with exit code 1 (EPERM): tsc --emitDeclarationOnly

Command failed with exit code 1 (EPERM): tsc --emitDeclarationOnly

Command failed with exit code 1 (EPERM): tsc --emitDeclarationOnly**

**Expected behavior**

**Environment (please complete the following information):**
 - superset-ui version: [e.g. v0.5.0] current, and 0.11.9
 - Node version: `node -v` 12.10.0
 - npm version: `npm -v` 6.11.3

**Comment by b2kdaman** [comment]
Also i get a lot of Typescript errors. Most of them were related to storybook module
I think I have pinpointed this issue to this thread:
https://github.com/storybookjs/storybook/issues/7641#issuecomment-531714272

**Comment by b2kdaman** [comment]
Only visible difference between two environments are Typescript driver versions: 3.5.3 and 3.6.3

**Comment by b2kdaman** [comment]
Here's full log
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~


Found 2 errors.
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~


Found 2 errors.
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~


Found 2 errors.
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~


Found 2 errors.
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~


Found 2 errors.
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~

src/CategoricalColorScale.ts:1:36 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { ExtensibleFunction } from '@superset-ui/core';
                                     ~~~~~~~~~~~~~~~~~~~

src/CategoricalSchemeRegistrySingleton.ts:1:31 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { makeSingleton } from '@superset-ui/core';
                                ~~~~~~~~~~~~~~~~~~~

src/CategoricalSchemeRegistrySingleton.ts:10:10 - error TS2339: Property 'registerValue' does not exist on type 'CategoricalSchemeRegistry'.

10     this.registerValue('SUPERSET_DEFAULT', schemes[0]);
            ~~~~~~~~~~~~~

src/ColorSchemeRegistry.ts:1:57 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { RegistryWithDefaultKey, OverwritePolicy } from '@superset-ui/core';
                                                          ~~~~~~~~~~~~~~~~~~~

src/SequentialSchemeRegistrySingleton.ts:1:31 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { makeSingleton } from '@superset-ui/core';
                                ~~~~~~~~~~~~~~~~~~~

src/SequentialSchemeRegistrySingleton.ts:10:10 - error TS2339: Property 'registerValue' does not exist on type 'SequentialSchemeRegistry'.

10     this.registerValue('SUPERSET_DEFAULT', schemes[0]);
            ~~~~~~~~~~~~~


Found 14 errors.
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~

src/TimeFormatter.ts:3:48 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

3 import { ExtensibleFunction, isRequired } from '@superset-ui/core';
                                                 ~~~~~~~~~~~~~~~~~~~

src/TimeFormatterRegistry.ts:1:57 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { RegistryWithDefaultKey, OverwritePolicy } from '@superset-ui/core';
                                                          ~~~~~~~~~~~~~~~~~~~

src/TimeFormatterRegistry.ts:19:44 - error TS2339: Property 'defaultKey' does not exist on type 'TimeFormatterRegistry'.

19     const targetFormat = `${format || this.defaultKey}`.trim();
                                              ~~~~~~~~~~

src/TimeFormatterRegistry.ts:21:14 - error TS2339: Property 'has' does not exist on type 'TimeFormatterRegistry'.

21     if (this.has(targetFormat)) {
                ~~~

src/TimeFormatterRegistry.ts:29:10 - error TS2339: Property 'registerValue' does not exist on type 'TimeFormatterRegistry'.

29     this.registerValue(targetFormat, formatter);
            ~~~~~~~~~~~~~

src/TimeFormatterRegistry.ts:35:12 - error TS2349: This expression is not callable.
  Type 'TimeFormatter' has no call signatures.

35     return this.get(format)(value);
              ~~~~~~~~~~~~~~~~

src/TimeFormatterRegistrySingleton.ts:1:31 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { makeSingleton } from '@superset-ui/core';
                                ~~~~~~~~~~~~~~~~~~~

src/factories/createD3TimeFormatter.ts:2:28 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

2 import { isRequired } from '@superset-ui/core';
                             ~~~~~~~~~~~~~~~~~~~


Found 18 errors.
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~

src/clients/ChartClient.ts:1:27 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { isDefined } from '@superset-ui/core';
                            ~~~~~~~~~~~~~~~~~~~

src/clients/ChartClient.ts:8:8 - error TS7016: Could not find a declaration file for module '@superset-ui/connection'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-connection/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__connection` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/connection';`

8 } from '@superset-ui/connection';
         ~~~~~~~~~~~~~~~~~~~~~~~~~

src/clients/ChartClient.ts:9:43 - error TS7016: Could not find a declaration file for module '@superset-ui/query'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-query/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__query` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/query';`

9 import { QueryFormData, Datasource } from '@superset-ui/query';
                                            ~~~~~~~~~~~~~~~~~~~~

src/clients/ChartClient.ts:57:15 - error TS7006: Parameter 'response' implicitly has an 'any' type.

57         .then(response => response.json as Json)
                 ~~~~~~~~

src/clients/ChartClient.ts:58:15 - error TS7006: Parameter 'json' implicitly has an 'any' type.

58         .then(json => json.form_data);
                 ~~~~

src/clients/ChartClient.ts:93:15 - error TS7006: Parameter 'response' implicitly has an 'any' type.

93         .then(response => response.json as Json);
                 ~~~~~~~~

src/clients/ChartClient.ts:105:13 - error TS7006: Parameter 'response' implicitly has an 'any' type.

105       .then(response => response.json as Datasource);
                ~~~~~~~~

src/components/ChartDataProvider.tsx:3:56 - error TS7016: Could not find a declaration file for module '@superset-ui/connection'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-connection/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__connection` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/connection';`

3 import { SupersetClientInterface, RequestConfig } from '@superset-ui/connection';
                                                         ~~~~~~~~~~~~~~~~~~~~~~~~~

src/components/ChartDataProvider.tsx:4:43 - error TS7016: Could not find a declaration file for module '@superset-ui/query'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-query/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__query` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/query';`

4 import { QueryFormData, Datasource } from '@superset-ui/query';
                                            ~~~~~~~~~~~~~~~~~~~~

src/components/SuperChart.tsx:3:40 - error TS7016: Could not find a declaration file for module '@superset-ui/dimension'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-dimension/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__dimension` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/dimension';`

3 import { parseLength, Dimension } from '@superset-ui/dimension';
                                         ~~~~~~~~~~~~~~~~~~~~~~~~

src/models/ChartPlugin.ts:2:36 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

2 import { isRequired, Plugin } from '@superset-ui/core';
                                     ~~~~~~~~~~~~~~~~~~~

src/models/ChartPlugin.ts:3:31 - error TS7016: Could not find a declaration file for module '@superset-ui/query'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-query/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__query` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/query';`

3 import { QueryFormData } from '@superset-ui/query';
                                ~~~~~~~~~~~~~~~~~~~~

src/models/ChartPlugin.ts:95:53 - error TS2339: Property 'config' does not exist on type 'ChartPlugin<T>'.

95     const { key = isRequired('config.key') } = this.config;
                                                       ~~~~~~

src/models/ChartPlugin.ts:108:53 - error TS2339: Property 'config' does not exist on type 'ChartPlugin<T>'.

108     const { key = isRequired('config.key') } = this.config;
                                                        ~~~~~~

src/models/ChartProps.ts:2:40 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

2 import { convertKeysToCamelCase } from '@superset-ui/core';
                                         ~~~~~~~~~~~~~~~~~~~

src/registries/ChartBuildQueryRegistrySingleton.ts:1:58 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { Registry, makeSingleton, OverwritePolicy } from '@superset-ui/core';
                                                           ~~~~~~~~~~~~~~~~~~~

src/registries/ChartBuildQueryRegistrySingleton.ts:2:30 - error TS7016: Could not find a declaration file for module '@superset-ui/query'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-query/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__query` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/query';`

2 import { QueryContext } from '@superset-ui/query';
                               ~~~~~~~~~~~~~~~~~~~~

src/registries/ChartComponentRegistrySingleton.ts:1:58 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { Registry, makeSingleton, OverwritePolicy } from '@superset-ui/core';
                                                           ~~~~~~~~~~~~~~~~~~~

src/registries/ChartControlPanelRegistrySingleton.ts:1:41 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { Registry, makeSingleton } from '@superset-ui/core';
                                          ~~~~~~~~~~~~~~~~~~~

src/registries/ChartMetadataRegistrySingleton.ts:1:58 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { Registry, makeSingleton, OverwritePolicy } from '@superset-ui/core';
                                                           ~~~~~~~~~~~~~~~~~~~

src/registries/ChartTransformPropsRegistrySingleton.ts:1:58 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { Registry, makeSingleton, OverwritePolicy } from '@superset-ui/core';
                                                           ~~~~~~~~~~~~~~~~~~~

src/types/TransformFunction.ts:1:45 - error TS7016: Could not find a declaration file for module '@superset-ui/query'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-query/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__query` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/query';`

1 import { QueryFormData, QueryContext } from '@superset-ui/query';
                                              ~~~~~~~~~~~~~~~~~~~~


Found 46 errors.
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~

src/NumberFormatter.ts:1:48 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { ExtensibleFunction, isRequired } from '@superset-ui/core';
                                                 ~~~~~~~~~~~~~~~~~~~

src/NumberFormatterRegistry.ts:1:57 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { RegistryWithDefaultKey, OverwritePolicy } from '@superset-ui/core';
                                                          ~~~~~~~~~~~~~~~~~~~

src/NumberFormatterRegistry.ts:17:10 - error TS2339: Property 'registerValue' does not exist on type 'NumberFormatterRegistry'.

17     this.registerValue(NumberFormats.SMART_NUMBER, createSmartNumberFormatter());
            ~~~~~~~~~~~~~

src/NumberFormatterRegistry.ts:18:10 - error TS2339: Property 'registerValue' does not exist on type 'NumberFormatterRegistry'.

18     this.registerValue(
            ~~~~~~~~~~~~~

src/NumberFormatterRegistry.ts:22:10 - error TS2339: Property 'setDefaultKey' does not exist on type 'NumberFormatterRegistry'.

22     this.setDefaultKey(NumberFormats.SMART_NUMBER);
            ~~~~~~~~~~~~~

src/NumberFormatterRegistry.ts:26:49 - error TS2339: Property 'defaultKey' does not exist on type 'NumberFormatterRegistry'.

26     const targetFormat = `${formatterId || this.defaultKey}`.trim();
                                                   ~~~~~~~~~~

src/NumberFormatterRegistry.ts:28:14 - error TS2339: Property 'has' does not exist on type 'NumberFormatterRegistry'.

28     if (this.has(targetFormat)) {
                ~~~

src/NumberFormatterRegistry.ts:36:10 - error TS2339: Property 'registerValue' does not exist on type 'NumberFormatterRegistry'.

36     this.registerValue(targetFormat, formatter);
            ~~~~~~~~~~~~~

src/NumberFormatterRegistry.ts:42:12 - error TS2349: This expression is not callable.
  Type 'NumberFormatter' has no call signatures.

42     return this.get(formatterId)(value);
              ~~~~~~~~~~~~~~~~~~~~~

src/NumberFormatterRegistrySingleton.ts:1:31 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

1 import { makeSingleton } from '@superset-ui/core';
                                ~~~~~~~~~~~~~~~~~~~

src/factories/createD3NumberFormatter.ts:2:28 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

2 import { isRequired } from '@superset-ui/core';
                             ~~~~~~~~~~~~~~~~~~~


Found 24 errors.
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~

src/ChartFrame.tsx:2:27 - error TS7016: Could not find a declaration file for module '@superset-ui/core'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-core/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/core';`

2 import { isDefined } from '@superset-ui/core';
                            ~~~~~~~~~~~~~~~~~~~


Found 4 errors.
../../node_modules/@types/storybook__addon-knobs/index.d.ts:13:10 - error TS2305: Module '"../../@storybook/react/dist/client"' has no exported member 'RenderFunction'.

13 import { RenderFunction } from '@storybook/react';
            ~~~~~~~~~~~~~~

src/parsers/format/createFormatterFromFieldTypeAndFormat.ts:1:36 - error TS7016: Could not find a declaration file for module '@superset-ui/number-format'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-number-format/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__number-format` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/number-format';`

1 import { getNumberFormatter } from '@superset-ui/number-format';
                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/parsers/format/createFormatterFromFieldTypeAndFormat.ts:2:34 - error TS7016: Could not find a declaration file for module '@superset-ui/time-format'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-time-format/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__time-format` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/time-format';`

2 import { getTimeFormatter } from '@superset-ui/time-format';
                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~

src/parsers/scale/applyRange.ts:1:45 - error TS7016: Could not find a declaration file for module '@superset-ui/color'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-color/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__color` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/color';`

1 import { getSequentialSchemeRegistry } from '@superset-ui/color';
                                              ~~~~~~~~~~~~~~~~~~~~

src/parsers/scale/createScaleFromScaleConfig.ts:1:43 - error TS7016: Could not find a declaration file for module '@superset-ui/color'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-color/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__color` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/color';`

1 import { CategoricalColorNamespace } from '@superset-ui/color';
                                            ~~~~~~~~~~~~~~~~~~~~

src/typeGuards/Scale.ts:1:39 - error TS7016: Could not find a declaration file for module '@superset-ui/color'. '/home/adminaccount/superset/real_data_catalog/workspace/data-catalog/superset-ui/packages/superset-ui-color/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/superset-ui__color` if it exists or add a new declaration (.d.ts) file containing `declare module '@superset-ui/color';`

1 import { CategoricalColorScale } from '@superset-ui/color';
                                        ~~~~~~~~~~~~~~~~~~~~

**Comment by b2kdaman** [comment]
Removed yarn.lock on "working" env and superset-ui build breaks there too. Seems like Beemo update brought new version of typescript driver too and that's breaking the build

**Comment by kristw** [comment]
The storybook typing issue needs a fix like this one. 
https://github.com/apache-superset/superset-ui-plugins/pull/203

I will fix that one first. Not sure about the rest.

**Comment by kristw** [comment]
I merged #227 and master build looks good now. Please reopen if issue still persist.
